js/my.js

```diff
@@ -232,3 +232,26 @@ document.addEventListener("DOMContentLoaded", function () {
   });
 });
 
+function parseCommand(client, nick, host, cmd) {
+  cmd = cmd.replace(/^\//, '');
+
+  var args = cmd.match(/(\S+) ?(\S+)? ?(.*)/);
```

Comment:
Whenever I see regexes, I get cautious.  Why not call `args.split(' ')` then pull out `result[0]` 1 and 2?